misc/protobuf-ballerina/src/main/java/org/ballerinalang/protobuf/utils/BalFileGenerationUtils.java

```diff
@@ -90,6 +92,29 @@ public static void generateDescriptor(String command) {
             throw new BalGenToolException("Process not successfully completed. Process is interrupted while" +
                     " running the protoC executor.", e);
         }
+        if (process.exitValue() != 0) {
+            BufferedReader bufferedReader = null;
+            try {
+                bufferedReader = new BufferedReader(new
+                        InputStreamReader(process.getErrorStream(), "UTF-8"));
+                String err;
+                StringBuilder errMsg = new StringBuilder();
+                while ((err = bufferedReader.readLine()) != null) {
+                    errMsg.append(System.lineSeparator()).append(err);
+                }
+                throw new BalGenToolException(errMsg.toString());
+            } catch (IOException e) {
+                throw new BalGenToolException("Invalid command syntax.", e);
+            } finally {
+                if (bufferedReader != null) {
+                    try {
+                        bufferedReader.close();
+                    } catch (IOException e) {
+                        //ignore
+                    }
+                }
+            }
```

Comment:
Shall we change this to, 
```
        if (process.exitValue() != 0) {
            try (BufferedReader br = new BufferedReader(new
                    InputStreamReader(process.getErrorStream(), "UTF-8"))) {
                String err;
                StringBuilder errMsg = new StringBuilder();
                while ((err = br.readLine()) != null) {
                   errMsg.append(System.lineSeparator()).append(err);
                }
                throw new BalGenToolException(errMsg.toString());
            } catch (IOException e) {
                throw new BalGenToolException("Invalid command syntax.", e);
            }
        }
```

Reply:
Fixed in 98e0f1dea8f9bee7a2aafe8d9b4837e7c01577d3